src/azure-cli/azure/cli/command_modules/vm/aaz/profile_2020_09_01_hybrid/ppg/_delete.py

```diff
@@ -89,7 +89,7 @@ def method(self):
 
         @property
         def error_format(self):
-            return "ODataV4Format"
+            return "MgmtErrorFormat"
```

Comment:
May I ask why the `error_format` will change? What impact will this change have on customers?

Reply:
It should be `2020-06-01` for the command in this profile, but it was mistakely set to `2022-11-01` in an old PR. Now this PR will correct the API version. I think the change of `error_format` is due to the downgrade of API version.

Reply:
MgmtErrorFormat is a more detailed error format than ODataV4Format. There's no breaking change.

Reply:
Got it, thanks